app/app.py

```diff
@@ -107,6 +114,47 @@ def get_prices(list):
     return prices_dict
 
 
+def get_time_series_prices(item):
+    api_codes = {
+        "Arabica Coffee": "COFFEE",
+        "Brent Crude Oil": "BRENTOIL",
+        "Cocoa": "COCOA",
+        "Corn": "CORN",
+        "Natural Gas": "NG",
+        "Canola": "CANO",
+        "Cotton": "COTTON",
+        "Oat": "OAT",
+        "Rice": "RICE",
+        "Soybeans": "SOYBEAN",
+        "Sugar": "SUGAR",
+        "Wheat": "WHEAT",
+        "Beef": "LCAT"
+    }
+    prices_dict = {}
+    today = datetime.today()
+    yesterday = (today - timedelta(days=1)).date()
+    lastMonthDate = get_date(today, 30)
+
+    url = "https://commodities-api.com/api/timeseries?access_key=0yw3m4s5g7fz8ialo49gqxtjnxbto11t59jg5qw4krdo15j48v37q06hgsnx&start_date=" + str(lastMonthDate) + "&end_date=" + str(yesterday) + "&symbols=" + \
+        api_codes[item]
+    response = requests.get(url)
+    jsonResponse = response.json()
+    prices_dict = jsonResponse["data"]["rates"]
+
+    return prices_dict, api_codes[item]
+
+
+def parse_data_for_graph(prices_dict, api_code):
+    labels = []
+    data = []
+
+    for date in prices_dict:
+        labels.append(date)
+        data.append(1 / prices_dict[date][api_code])
```

Comment:
@shreya2902 Do you mind confirming that I did this correctly with taking the inverse?

Reply:
Yes, that looks perfect!

Reply:
Yeah, that looks perfect!